teradatalabs/cdh5-hive-kerberized/files/conf/yarn-site.xml

```diff
@@ -15,7 +15,6 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
```

Comment:
It seems most our config files don't contain this line while the books on hadoop have it in config excertps. So I assume this is redundant.

Reworded the commits to `Remove redundant <xml-stylesheet/> declaration (...)`